test/functional/apps/dashboard/dashboard_time_picker.js

```diff
@@ -72,5 +73,16 @@ export default function ({ getService, getPageObjects }) {
       expect(time.end).to.be('Nov 17, 2015 @ 18:01:36.621');
       expect(refresh.interval).to.be('2');
     });
+
+    it('Timepicker respects dateFormat from UI settings', async () => {
+      await kibanaServer.uiSettings.replace({ 'dateFormat': 'YYYY-MM-DD HH:mm:ss.SSS', });
```

Comment:
Should this test suite set the format back to the default in the after method?  Seems like it might break other tests if this test leaves it modified?

Reply:
True, if more tests are added to this file. I'll add an `after` method

Reply:
I was thinking of other tests outside of this suite, but as long as new tests all load a new .kibana index (which they should), they'll lose this setting anyway.  So I'm OK with not adding an `after` method here.  But it would be safest if you add it.